app/src/components/Register.tsx

```diff
@@ -98,167 +116,164 @@ class Register extends React.Component<RegisterProps, RegisterState> {
+    const key = event.target.id;
+    const val = event.target.value;
+    const stateUpdate = { fields: this.state.fields as Pick<RegisterFormFields, keyof RegisterFormFields> };
+    stateUpdate.fields[key] = val;
+    this.setState(stateUpdate);
+  }
+
+  renderArtifactInformation = (): React.ReactNode => {
+    return (
+      <Container>
+        <Form.Row>
+          <Form.Group as={Col} controlId="title">
+            <Form.Label>Title</Form.Label>
+            <Form.Control
+              required
+              type="text"
+              onChange={this.inputChangeHandler}/>
+            <Form.Control.Feedback>Looks good!</Form.Control.Feedback>
+          </Form.Group>
+        </Form.Row>
+
+        <Form.Row>
+          <Form.Group as={Col} controlId="createdDate">
+            <Form.Label>Date of creation</Form.Label>
+            <Form.Control
+              required
+              type="text"
+              onChange={this.inputChangeHandler}/>
+            <Form.Control.Feedback>Looks good!</Form.Control.Feedback>
+          </Form.Group>
+        </Form.Row>
+
+        <Form.Row>
+          <Form.Group as={Col} controlId="medium">
+            <Form.Label>Medium of Artwork</Form.Label>
+            <Form.Control
+              required
+              type="text"
+              onChange={this.inputChangeHandler}/>
+            <Form.Control.Feedback>Looks good!</Form.Control.Feedback>
+          </Form.Group>
+
+          <Form.Group as={Col} controlId="size">
+            <Form.Label>Size of Creation</Form.Label>
+            <Form.Control
+              required
+              type="text"
+              onChange={this.inputChangeHandler}/>
+            <Form.Control.Feedback>Looks good!</Form.Control.Feedback>
+          </Form.Group>
+        </Form.Row>
+      </Container>
+    );
+  }
+
+  renderArtistInformation = (): React.ReactNode => {
+    return (
+      <Container>
+        <Form.Row>
+          <Form.Group as={Col} controlId="artistName">
+            <Form.Label>Artist Name</Form.Label>
+            <Form.Control
+              required
+              type="text"
+              onChange={this.inputChangeHandler}/>
+            <Form.Control.Feedback>Looks good!</Form.Control.Feedback>
+          </Form.Group>
+        </Form.Row>
+        <Form.Row>
+          <Form.Group as={Col} controlId="artistNationality">
+            <Form.Label>Artist Nationality</Form.Label>
+            <Form.Control
+              required
+              type="text"
+              onChange={this.inputChangeHandler}/>
+            <Form.Control.Feedback>Looks good!</Form.Control.Feedback>
+          </Form.Group>
+          <Form.Group as={Col} controlId="artistBirthYear">
+            <Form.Label>Artist Birth Year</Form.Label>
+            <Form.Control
+              required
+              type="text"
+              onChange={this.inputChangeHandler}/>
+            <Form.Control.Feedback>Looks good!</Form.Control.Feedback>
```

Comment:
I'm seeing "Looks good!" alot, maybe worth a variable, maybe not

Reply:
Probably, got a tad lazy with my feedback stuff.